src/node.h

```diff
@@ -86,6 +86,22 @@ struct cmark_node {
 
 CMARK_EXPORT int cmark_node_check(cmark_node *node, FILE *out);
 
+static inline bool CMARK_NODE_TYPE_BLOCK_P(cmark_node_type node_type) {
+	return node_type < CMARK_NODE_TEXT;
```

Comment:
Note that cmark.h defines CMARK_NODE_LAST_BLOCK and CMARK_NODE_FIRST_BLOCK, as well as CMARK_NODE_FIRST_INLINE and CMARK_NODE_LAST_INLINE.  So an easy way to check for blocks is
```c
node->type >= CMARK_NODE_FIRST_BLOCK && node->type <= CMARK_NODE_LAST_BLOCK
```
etc.  This is easy enough that I don't think it's really necessary to create these functions.

Reply:
I was trying to establish some level of consistency with the interface introduced by `cmark-gfm`.

Reply:
Yes, it may be a good idea in the end.  But I'd prefer to keep that issue separate from this change.